src/components/dashboard/index.js

```diff
@@ -144,14 +325,70 @@ const cleanUsername = (username) => {
     return user;
 };
 
-const cleanDescription = (description) => {
-    let desc;
-    if (description.length > constants.DESC_MAX_LENGTH) {
-        desc = description.slice(0, constants.DESC_MAX_LENGTH) + "...";
+const cleanField = (field, comparitor) => {
+    let retval;
+    if (field.length > comparitor) {
+        retval = field.slice(0, comparitor) + "...";
     } else {
-        desc = description;
+        retval = field;
     }
-    return desc;
+    return retval;
+};
+
+const cleanDescription = (description) =>
+    cleanField(description, constants.DESC_MAX_LENGTH);
+
+const cleanTitle = (title, isLarge = true) => {
+    let comparitor;
+
+    if (isLarge) {
+        comparitor = constants.TITLE_MAX_LENGTH;
+    } else {
+        comparitor = constants.TITLE_MAX_LENGTH_SMALL;
+    }
+
+    return cleanField(title, comparitor);
```

Comment:
This comes down to preference, but just a reminder that this type of simple conditional could also be written with the `? :` ternary syntax